tests/serialize/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The npm team still hasnt given any definitive response about how to make this less ugly
https://github.com/npm/npm/pull/10958

But the technique appear to be used here and there
 http://stackoverflow.com/questions/32783885/is-it-possible-to-use-npm-to-run-scripts-in-multiple-subfolders

http://stackoverflow.com/questions/31773546/the-best-way-to-run-npm-install-for-nested-folders

But moving everthing isn't much of a problem. The other advantage of this format is that the single testing scripts aren't accessible from the main folder, and it will be cleaner to migrate them to the mocha testing later.